include/net/zzhc.rst

```diff
@@ -0,0 +1,74 @@
+.. _lib_zzhc:
+
+China Telecom ZZHC library
+#################################################
+
+The China Telecom ZZHC library implements the self-registration (Zi-ZHu-Ce, ZZHC) functionality that registers the module information on the server when the network is available.
+
+Overview
+********
+
+The telecommunication modules that are part of the China Telecom network are required to provide certain information stored on the module to the `zzhc.vnet.cn` server.
+This is required for conformity with Chapter 6 of the China Telecom IoT Module Requirements Whitepaper.
+
+The ZZHC library is started as a background thread on boot and it automatically detects the availability of the operator network.
+Once the Internet connectivity is ready and a new SIM card is detected on boot, the library proactively uploads the following information to the ZZHC server:
+- IMEI
+- model number (that is, nRF9160)
+- modem revision
+- nRF9160 SIM ICCID
+- nRF9160 SIM IMSI
+On any failure, which include unknown/negative response from server or any error during program execution, the ZZHC library retries the next hour for at most 5 times.
+
+.. _lib_zzhc_dependencies:
+
+Dependencies
```

Comment:
Given that most of the library documentation does not feature a Dependencies section, I've moved this under Configuration.

Reply:
Yes, thanks.